vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -154,3 +154,320 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
+
+
+## Changing Selectors
+
+If you want to have a different selector for dashboard navigation you have to provide the `selector` argument of the [`Navigation`][vizro.models.Navigation] with a different selector.
+Currently available selectors are [`NavBar`][vizro.models.NavBar] and [`Accordion`][vizro.models.Accordion]. If `selector` is not defined it defaults to `Accordion`.
+
+If you want to add icons to your dashboard, you have to provide [`NavBar`][vizro.models.NavBar] model
+
+   ```
+    Dashboard(
+        pages=[page_1, page_2],
+        navigation=Navigation(
+            pages=["My first page", "My second page"]
+            selector=NavBar()
+        )
+    )
+   ```
+
+!!! example "Adding navigation icons"
+    === "app.py"
+        ```py
+        from vizro import Vizro
+        import vizro.plotly.express as px
+        import vizro.models as vm
+
+        iris = px.data.iris()
+
+        page_1 = vm.Page(
+                    title="My first page",
+                    components=[
+                        vm.Graph(id="scatter_chart", figure=px.scatter(iris, x="sepal_length", y="petal_width", color="species")),
+                    ],
+        )
+        page_2 = vm.Page(
+                    title="My second page",
+                    components=[
+                        vm.Graph(id="line_chart", figure=px.line(iris, x="sepal_length", y="petal_width", color="species")),
+                    ],
+                )
+
+        dashboard = vm.Dashboard(
+                        pages=[page_1, page_2],
+                        navigation=vm.Navigation(pages=["My first page", "My second page"], selector=vm.NavBar())
+                    )
+
+        Vizro().build(dashboard).run()
+        ```
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: scatter
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: scatter_chart
+                type: graph
+            title: My first page
+        - components:
+             - figure:
+                  _target_: line
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: line_chart
+                type: graph
+            title: My second page
+        navigation:
+            pages:
+             First title:
+                - My first page
+             Second title:
+                - My second page
+            selector:
+                type: navbar
+
+        ```
+    === "Result"
+        [![NavigationIcons]][NavigationIcons]
+
+    [NavigationIcons]: ../../assets/user_guides/navigation/navigation_icons.png
+
+
+## Further customization
```

Comment:
After reading this entirely, I would propose a different structure. I think the problem with the current one is that as a reader I am quickly lost for a) where I have to customize, and b) what I can customize (see also comment on intro of NavItem)

I would propose the following. Let's create a section called `Customizing selectors` (after `Changing selectors`) where we can quickly introduce the idea that instead(is this true?) of providing the pages argument to the `Navigation` model, you can also provide the pages to your selector alongside more customization.

Then I would go down a section hierarchy and would have headers: `Accordion` and `NavBar`.

Then we can always have a simple structure: short intro, your nice config example(s) in tabular form, not much more.

Under `Accordion`, this is probably just the simple example where you add the pages to the `Accordion` instead of the `Navigation` right?

Under `NavBar`, the intro could be a bit longer saying here you cannot just add the pages, but you have to add `NavItems`. Maybe! a very small intro of the `NavItems` arguments, but potentially the examples will speak for itself. Then I would keep it shorter than what you have below and do all examples in tabular form. I think the user will get how to do it, but we should have meaningful titles for the different options, so its searchable.

Does that make sense?